Aufgabe: Pool A: Crafting (#275)
PM-Dungeon/PM-Lecture: markdown/assignments/pool_a/crafting/index.md

hidden: true
---

## Ziel

> In dieser Aufgabe sollen Sie ein Crafting-System implementieren. Also eine Möglichkeit für den Helden, aus verschiedenen Gegenständen neue Gegenstände herzustellen. Denken Sie an das brauen von Tränken oder schmieden von Waffen.

Reviewer: ```suggestion
In dieser Aufgabe sollen Sie ein Crafting-System implementieren. Das ist eine Möglichkeit für den Helden, aus verschiedenen Gegenständen nach bestimmten Regeln neue Gegenstände herstellen zu können. Denken Sie beispielsweise an das Brauen von Tränken oder Schmieden von Waffen.
```

## Voraussetzung

> Um diese Aufgabe lösen zu können, müssen Sie vorher die Aufgabe `[Inventar]({{<ref "/assignments/pool_a/inventar_text_based" >}})`{=markdown} gelöst haben.

Reviewer: ```suggestion
Um diese Aufgabe lösen zu können, müssen Sie vorher die Aufgabe `["Inventar"]({{< ref "/assignments/pool_a/inventar_text_based" >}})`{=markdown} gelöst haben.
```

## Crafting

Implementieren Sie ein Crafting-System.
Lassen Sie den Helden mindestens einen Trank, eine Waffe und einen Zauberspruch craften.

> Um Items zu craften muss vorher eine Rezeptur, entsprechendes Werkzeug sowie die Zutaten gefunden werden.

Reviewer: ```suggestion
Um Items zu craften müssen vorher eine Rezeptur, entsprechendes Werkzeug sowie die nötigen Zutaten gefunden werden.
```
